tests/.pest/snapshots/DenmarkTest/it_can_calculate_danish_holidays.snap

```diff
@@ -0,0 +1,46 @@
+[
+    {
+        "name": "Nyt\u00e5r",
+        "date": "2024-01-01"
+    },
+    {
+        "name": "Sk\u00e6rtorsdag",
+        "date": "2024-03-28"
+    },
+    {
+        "name": "Langfredag",
+        "date": "2024-03-29"
+    },
+    {
+        "name": "P\u00e5skedag",
+        "date": "2024-04-01"
+    },
+    {
+        "name": "Anden P\u00e5skedag",
+        "date": "2024-04-02"
+    },
+    {
+        "name": "Kristi Himmelfartsdag",
+        "date": "2024-05-09"
+    },
+    {
+        "name": "Pinse",
+        "date": "2024-05-19"
+    },
+    {
+        "name": "Anden Pinsedag",
+        "date": "2024-05-20"
+    },
+    {
+        "name": "Juleaften",
+        "date": "2025-12-12"
```

Comment:
Could you verify all generated results? It seems like something is not right yet.

Reply:
Sorry! Should be correct now.